tests/s3/test_multipart_getput.py

```diff
@@ -316,19 +322,28 @@ def test_multipart_upload_test_28538(self):
         self.log.info("Listed parts of multipart upload: %s", res[1])
         self.log.info(" Complete the multipart with first and last part upload")
         uploaded_parts[1].reverse()
-        try:
-            resp = self.s3_mpu_test_obj.complete_multipart_upload(mpu_id, uploaded_parts[1],
+        if S3_ENGINE_RGW == CMN_CFG["s3_engine"]:
+            try:
+                resp = self.s3_mpu_test_obj.complete_multipart_upload(mpu_id, uploaded_parts[1],
                                                                   self.bucket_name,
                                                                   self.object_name)
-            assert_utils.assert_false(resp[0], resp[1])
-        except CTException as error:
-            self.log.error(error)
-            assert_s3_err_msg(errmsg.RGW_ERR_WRONG_JSON, errmsg.CORTX_ERR_WRONG_JSON,
-                              CMN_CFG["s3_engine"], error)
-            self.log.info("Failed to complete the multipart with incomplete part details ")
-        self.log.info("Aborting multipart uploads")
-        self.s3_mpu_test_obj.abort_multipart_upload(self.bucket_name,
-                                                    self.object_name, mpu_id)
+                assert_utils.assert_true(resp[0], resp[1])
+            except CTException as error:
+                self.log.error(error)
```

Comment:
If you run into exception in this except,  you would break the behavior of test.

Reply:
I modified this , and as we discussed, will handle the RGW related filtering in next PI